imap/cyr_expire.c

```diff
@@ -650,22 +849,20 @@ static int parse_args(int argc, char *argv[], struct arguments *args)
 int main(int argc, char *argv[])
 {
     int r = 0;
-    struct arguments args;
+    struct cyr_expire_ctx ctx;
```

Comment:
I did want to do that (assigning to `{0}` is my preferred method to initialise, but didn't see it being used in cyrus, so chose the `memset` way :). More than happy to change!